### DISCUSSION TITLE
设置单点登录绑定微软账户失败，提示如下

### DISCUSSION THREAD

**Discussion by ironstraight** [AlistGo/alist#4380] [Q&A]
根据教程设置，绑定微软账户时绑定失败提示如下：
AADSTS50011: The redirect URI 'http://127.0.0.1:5244/api/auth/sso_callback?method=get_sso_id' specified in the request does not match the redirect URIs configured for the application 'e9e4e8d5-600c-4f87-b3ae-f4372cd04acc'. Make sure the redirect URI sent in the request matches one added to your application in the Azure portal. Navigate to https://aka.ms/redirectUriMismatchError to learn more about how to fix this.

已确认密钥和客户端ID无错都是按照教程设置，我是本地搭建的，并非客户端。
<img width="749" alt="屏幕截图 2023-05-13 210358" src="https://github.com/alist-org/alist/assets/100616725/b01e1b8c-cdbe-43d1-9e77-28439904bda6">

**Comment by anwen-anyi** [comment]
1. 截图你微软应用程序的回调参数配置页面（检查是否正确）
   - 一共需要两个
2. 检查你是否写错了应用的客户端ID和秘钥
3. 你是在本地搭建的还是服务器？
   - 有服务器不用写 127.0.0.1:5244
      - 如果是服务器否反向代理了

教程没问题，反复验证了好几次的

**Reply by ironstraight** [reply]
已更新。请参阅截图。已确认密钥和客户端ID无错都是按照教程设置，我是本地搭建的，并非客户端。
